src/app/_be/organizations/Organization.js

```diff
@@ -46,94 +46,72 @@ export default class Organization {
 
     constructor(orgID = null){
```

Comment:
This class needs a method to set the organization uid (If not provided initially), and populates the organization class with the correct organization data from the database